src/lib/transition_router/transition_router.ml

```diff
@@ -13,8 +13,8 @@ type Structured_log_events.t += Starting_transition_frontier_controller
 type Structured_log_events.t += Starting_bootstrap_controller
   [@@deriving register_event {msg= "Starting bootstrap controller phase"}]
 
-let create_bufferred_pipe ?name () =
-  Strict_pipe.create ?name (Buffered (`Capacity 50, `Overflow Crash))
+let create_bufferred_pipe ?(capacity = 50) ?name () =
```

Comment:
Do we need to worry here about saturating the bootstrap/transition frontier controller pipes, and then overflowing the downstream pipes? It seems as though we might now allow more blocks to enter the system than the later pipes have capacity to handle.

In particular, could you generate 100 distinct blocks by re-ordering transactions within a single valid block's diff, and use that to trigger this crash? Perhaps it would be safer here to `Overflow Drop_head`?

Reply:
made a flow diagram of pipes and i think it makes sense just to drop head on overflow. Updated the description of the PR accordingly